packages/perry-solid/test/native-smoke.py

```
@@ -0,0 +1,105 @@
+        assert json.load(response)["ok"]
+
+
+def capture(name):
+    tree = get("/widgets?tree=true")
+    (args.output_dir / f"{name}.json").write_bytes(tree)
+    (args.output_dir / f"{name}.png").write_bytes(get("/screenshot"))
+    return json.loads(tree)
+
+
+with (args.output_dir / "stdout.log").open("wb") as stdout, (args.output_dir / "stderr.log").open("wb") as stderr:
+    process = subprocess.Popen([str(args.binary.resolve())], stdout=stdout, stderr=stderr)
```

Ruff (0.16.3)

[error] `subprocess` call: check for execution of untrusted input

(S603)